ansible/testbed-cli.sh

```diff
@@ -116,6 +117,21 @@ function renumber_topo
   echo Done
 }
 
+function add_config
```

Comment:
it is actually a part of the testbed configuration. I think it is better to have everything in one place, like  
```
    testbed-cli.sh start-vms ...
    testbed-cli.sh add-topo ...
    testbed-cli.sh add-config ...
```